testrtc.py

```diff
@@ -21,7 +27,42 @@ def get(self, size_kbytes):
     self.response.headers.add_header("Access-Control-Allow-Origin", "*")
     self.response.headers['Content-Type'] = 'application/octet-stream'
     self.response.out.write(random_file[0: int(size_kbytes)*1000])
-    
+
+class NewReportHandler(blobstore_handlers.BlobstoreUploadHandler):
+  # Generate and return blobstore upload URL.
+  def head(self):
+    upload_url = blobstore.create_upload_url('/report/new')
+    self.response.headers['Response-Text'] = upload_url
+
+  # On uploaded file to the blobstore.
+  def post(self):
+    origin = self.request.headers['Origin']
+    fileName = self.request.headers['X-File-Name']
+    upload_files = self.get_uploads(field_name=fileName)
+    blob_info = upload_files[0]
+    self.response.headers['Response-Text'] = origin + '/report/%s' % blob_info.key()
+
+class ReportHandler(blobstore_handlers.BlobstoreDownloadHandler):
+  def get(self, resource):
```

Comment:
Can you have an explicit:
 self.response.headers['Content-Type'] = 'text/plain'

Don't want us to serve anything else from here.